F/FixedEffects/Compat.toml

```diff
@@ -1,8 +1,11 @@
-["0.1"]
+["0.1.0-0.1.1"]
 CategoricalArrays = "0.0-0.5"
 Reexport = "0.0-0.2"
 StatsBase = "0.1-0.29"
 julia = "1.0-1.3"
 
 ["0.1.1"]
 FillArrays = "0.0-0.5"
+
+["0.1.2"]
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.